README.md

```diff
@@ -1,14 +1,27 @@
 # HarbourBridge: Turnkey Postgres-to-Spanner Evaluation
-[![cloudspannerecosystem](https://circleci.com/gh/cloudspannerecosystem/harbourbridge.svg?style=svg)](https://circleci.com/gh/cloudspannerecosystem/harbourbridge)
 
-HarbourBridge is a stand-alone open-source tool for Cloud Spanner evaluation,
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
 using data from an existing PostgreSQL database. The tool ingests pg_dump
 output, automatically builds a Spanner schema, and creates a new Spanner
-database populated with data from pg_dump. The tool is designed to simplify
-Spanner evaluation. It is not intended for production database migration. In
-particular, statements and features in the pg_dump output that don't map
-directly on Spanner features are simply ignored. It is also not intended for
-databases larger than a couple of GB.
+database populated with data from pg_dump.
+
+HarbourBridge is designed to simplify Spanner evaluation, and in particular to
+bootstrap the process by getting moderate-size PostgreSQL datasets into Spanner
+(up to a few GB). Many features of PostgreSQL, especially those that don't map
+directly to Spanner features, are ignored e.g. (non-primary) indexes, functions
```

Comment:
A few extra commas?
"are ignored, e.g. (non-primary) indexes, functions, and sequences"